modules/4_mnist/src/mnist.cpp

```diff
@@ -56,15 +56,15 @@ float validate(Ptr<ml::KNearest> model,
 }
 
 int predict(Ptr<ml::KNearest> model, const Mat& image) {
-    // TODO: resize image to 28x28
+        // TODO: resize image to 28x28 (cv::resize)
+        
+        // TODO: convert image from BGR to HSV (cv::cvtColor)
```

Comment:
Please fix indentation (try to fix it in Visual Studio so tab are replaced by whitespaces).